packages/docs/src/examples/v-virtual-scroll/renderless.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-app>
+    <v-container>
+      <v-table>
+        <thead>
+          <tr>
+            <th>Col 1</th>
+            <th>Col 2</th>
+          </tr>
+        </thead>
+        <tbody>
+          <v-virtual-scroll :items="items" renderless>
```

Comment:
When I inspect the example in the DevTools I see 1000 nodes in the DOM. Does the feature even work?